meerkat_frontend/templates/reports/report_ctc.html

```diff
@@ -2,7 +2,7 @@
 {% block title %}{{report.data.project_region|e}} &middot; {{ _('Cholera Treatment Centre Report') }} &middot; {{ _('Week') }} {{ report.data.epi_week_num }}{% endblock title %}
 
 {% macro format_percent(input) %}
-  {{ input.Y }} ({{ (input.Y / input.N * 100 )|int}} %)
+  {{ input.Y }} ({{ (input.Y / input.N * 100 )|int if input.N > 0 else "N/A"}} %)
```

Comment:
I agree that in general it's better for the code to be concise, but given that the templates have no testing, I would advise to use longer but clearer logic. It's just a comment of course, no need to change that.